src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/CreateAccount/CreateAccountCommandHandler.cs

```diff
@@ -64,32 +66,35 @@ public class CreateAccountCommandHandler : IAsyncRequestHandler<CreateAccountCom
 
             var user = await GetUser(message);
 
-            var emprefs = message.PayeReference.Split(',');
 
             if (string.IsNullOrEmpty(message.OrganisationReferenceNumber))
             {
                 message.OrganisationReferenceNumber = Guid.NewGuid().ToString();
             }
 
-            var returnValue = await _accountRepository.CreateAccount(user.Id, message.OrganisationReferenceNumber, message.OrganisationName, message.OrganisationAddress, message.OrganisationDateOfInception, emprefs[0], message.AccessToken, message.RefreshToken, message.OrganisationStatus, message.EmployerRefName, (short)message.OrganisationType, message.PublicSectorDataSource, message.Sector);
+            var returnValue = await _accountRepository.CreateAccount(user.Id, message.OrganisationReferenceNumber, message.OrganisationName, message.OrganisationAddress, message.OrganisationDateOfInception, message.PayeReference, message.AccessToken, message.RefreshToken, message.OrganisationStatus, message.EmployerRefName, (short)message.OrganisationType, message.PublicSectorDataSource, message.Sector);

```

Comment:
thinking returnValue could be name something better now that we are changing the code

Reply:
createAccountResult instead